training/src/main/scala/com/airbnb/aerosolve/training/pipeline/GenericPipeline.scala

```diff
@@ -48,6 +48,20 @@ object GenericPipeline {
       .saveAsTextFile(output, classOf[GzipCodec])
   }
 
+  def makeTestingRun(sc: SparkContext, config: Config) = {
```

Comment:
This is actually doing the same thing as `makeTrainingRun` it's just the config is different. I feel we can have some thing like `GenericPipeline.makeExampleRun(sc, config)`, then we can share some code.